test/sql_tests.cpp

```diff
@@ -40,27 +43,75 @@ TEST(CreateStatementTest) {
 
   const CreateStatement* stmt = (const CreateStatement*)result.getStatement(0);
   ASSERT_EQ(stmt->type, kCreateTable);
-  ASSERT_STREQ(stmt->tableName, "students");
+  ASSERT_STREQ(stmt->tableName, "dummy_table");
   ASSERT_NOTNULL(stmt->columns);
-  ASSERT_EQ(stmt->columns->size(), 5);
-  ASSERT_STREQ(stmt->columns->at(0)->name, "name");
-  ASSERT_STREQ(stmt->columns->at(1)->name, "student_number");
-  ASSERT_STREQ(stmt->columns->at(2)->name, "city");
-  ASSERT_STREQ(stmt->columns->at(3)->name, "grade");
-  ASSERT_STREQ(stmt->columns->at(4)->name, "comment");
-  ASSERT_EQ(stmt->columns->at(0)->type, (ColumnType{DataType::VARCHAR, 50}));
-  ASSERT_EQ(stmt->columns->at(1)->type, ColumnType{DataType::INT});
-  ASSERT_EQ(stmt->columns->at(2)->type, ColumnType{DataType::INT});
-  ASSERT_EQ(stmt->columns->at(3)->type, ColumnType{DataType::DOUBLE});
-  ASSERT_EQ(stmt->columns->at(4)->type, ColumnType{DataType::TEXT});
+  ASSERT_EQ(stmt->columns->size(), 18);
+  ASSERT_STREQ(stmt->columns->at(0)->name, "c_char");
```

Comment:
I would prefer to order the checks by columns rather than by the type of the check.
This would help me to understand directly, which column is at position 11.
Example:
ASSERT_STREQ(stmt->columns->at(0)->name, "c_char");
ASSERT_EQ(stmt->columns->at(0)->type, (ColumnType{DataType::CHAR, 42}));
ASSERT_NEQ(stmt->columns->at(0)->type, (ColumnType{DataType::CHAR, 43}));
ASSERT_EQ(stmt->columns->at(0)->nullable, false);

ASSERT_STREQ(stmt->columns->at(1)->name, "c_date");
...